.goreleaser.yml

```diff
@@ -0,0 +1,132 @@
+before:
+  hooks:
+    - go version | grep --quiet "go1\.15\.2" || echo "Go binary version must be 1.15.2"
+    - go mod download
+builds:
+  - id: operator-sdk
+    main: ./cmd/operator-sdk
+    binary: operator-sdk
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    asmflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    gcflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
+      - -X {{ .Env.REPO }}/internal/version.KubernetesVersion={{ .Env.K8S_VERSION }}
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
+    goos:
+      - linux
+      - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
```

Comment:
`targets` can be used here instead. We can also use yaml anchors to decrease duplication. Something like:

```yaml
targets: &build-targets
  - darwin_amd64
  - linux_amd64
  - linux_arm64
  - linux_ppc64le
  - linux_s390x
```

And then subsequent builds for helm-operator and ansible-operator can then just do:
```yaml
targets: *build-targets
```

Same for other fields we want to reuse.